decentraland/screens/MyAccountScreen.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import MyAccount from "@presentational/MyAccount";
+import { connect } from "react-redux";
+import AccountCreationActions from "@constants/AccountCreationActions";
+import ActionStatus from "../constants/ActionStatus";
+import PropTypes from "prop-types";
+
+export class MyAccountScreen extends React.Component {
+  static navigationOptions = {
+    title: "My Account",
```

Comment:
Is this showing up at the top of the screen? We'll either want to consistently use this for all screens or not.

Reply:
```suggestion
    title: "My account",
```

Reply:
> Is this showing up at the top of the screen? We'll either want to consistently use this for all screens or not.

Noting that it is showing up at the top of the screen after I tried it out locally

Reply:
Yup, I've put it there because it was in mockup you've sent. I can remove but I would suggest that you use it on all screen, it's more user friendly I think

Reply:
Okay, it's fine to keep it in on this screen. I'm not sure it looks good on an infinite scroll screen like the `Land for sale` one, though. But I think it's fine to have it on some screens but not others.

Reply:
@mpetrunic Feel free to accept the suggestion above via the GitHub UI. Thanks!